lib/ymlr/encoder_test.exs

```diff
@@ -199,7 +199,11 @@ defmodule Ymlr.EncoderTest do
     end
 
     test "datetime" do
-      assert MUT.to_s!(DateTime.from_naive!(~N[2016-05-24 13:26:08.003], "Etc/UTC")) == "2016-05-24 13:26:08.000000000 Z"
+      assert MUT.to_s!(~U[2016-05-24 13:26:08Z])      == "2016-05-24T13:26:08Z"
+      assert MUT.to_s!(~U[2016-05-24 13:26:08.1Z])    == "2016-05-24T13:26:08.1Z"
+      assert MUT.to_s!(~U[2016-05-24 13:26:08.02Z])   == "2016-05-24T13:26:08.02Z"
+      assert MUT.to_s!(~U[2016-05-24 13:26:08.003Z])  == "2016-05-24T13:26:08.003Z"
+      assert MUT.to_s!(~U[2016-05-24 13:26:08.0004Z]) == "2016-05-24T13:26:08.0004Z"
```

Comment:
Wanna add a non-UTC test case?

Reply:
DateTime only supports UTC, we would have to add a dependency to `tz` or `tzdata`, see https://hexdocs.pm/elixir/1.13/DateTime.html#module-time-zone-database

(i.e. I try to do exactly that and the compiler would not let me)